backend/init_data/skills/pptx/__init__.py

```
@@ -0,0 +1,10 @@
+# SPDX-FileCopyrightText: 2025 Weibo, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""PPTX skill package."""
+
+from .provider import PPTXToolProvider
+from .pptx_tool import PPTXGenerateTool
+
+__all__ = ["PPTXToolProvider", "PPTXGenerateTool"]
```

Ruff (0.14.13)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)